src/cpu/o3/FuncScheduler.py

```
@@ -85,7 +90,7 @@ class IssueQue(SimObject):
     inports = Param.Int(2, "")
     scheduleToExecDelay = Param.Cycles(2, "")
     oports = VectorParam.IssuePort("")
-    sel = Param.BaseSelector(BaseSelector(), "Selector for this IQ (default: age first)")
+    sel = Param.BaseSelector(SMTBasedSelector(), "Selector for this IQ (default: age first)")
```

Ruff (0.15.5)

[error] `Param` may be undefined, or defined from star imports

(F405)